### DISCUSSION TITLE
English and German training data❓ Questions / Help / Support

### DISCUSSION THREAD

**Discussion by tugstugi** [snakers4/silero-models#19] [General]
On which datasets were the released English and German CE models trained?

thanks

**Comment by snakers4** [comment]
Hi,

We train them on proprietary datasets collected by us.

**Comment by tugstugi** [comment]
thanks for info. I have trained your CE model on LibriSpeech with the native PyTorch CTC loss and the novograd optimizer on mel spectrograms, it can reach in 100 GPU hours around test-clean 9% WER and test-other 22% WER. It sounds reasonable if your CE models were trained on proprietary datasets.

**Comment by snakers4** [comment]
> I have trained your CE model on LibriSpeech

note that you can essentially view ML models as compression algorithms
and the smaller the dataset / domain - the more the model overfits
therefore we try to report metrics on dozens of datasets, because librispeech alone is just easily abused

I especially like this chart adapted from [here](https://paperswithcode.com/sota/speech-recognition-on-librispeech-test-other)
![image](https://user-images.githubusercontent.com/12515440/94804017-d70f3880-03f2-11eb-80d1-b52592c671c0.png)

you can get better results if you limit the domain (one common trick everybody abuses - is training huge domain-limited LMs that are orders of magnitude large than text corpora themselves), but this is no the purpose of our project

our purpose is to pack all the knowledge into one compact generalist model which is difficult
